cmake/External_cactus.cmake

```diff
@@ -24,12 +24,14 @@ ExternalProject_Add(cactus-amc13
   PATCH_COMMAND patch -p1 -i "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/amc13-cpp11.patch"
   COMMAND       patch -p1 -i "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/amc13-python3.patch"
   COMMAND       patch -p1 -i "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/amc13-install.patch"
+  COMMAND       patch -p1 -i "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/amc13-as-needed.patch"
 
   CONFIGURE_COMMAND ""
   BUILD_IN_SOURCE TRUE
   BUILD_COMMAND make
     "CACTUS_ROOT=${CMAKE_BINARY_DIR}/install/cactus"
     "INSTALL_PREFIX=<INSTALL_DIR>"
+    "SHELL=bash"
```

Comment:
Have you tried you your patch in a clean build ? I'm a bit surprised that you don't need to pass `"SHELL=bash"` to the `INSTALL_COMMAND`. Or maybe that the directories are already created by `cactus-ipbus` ?